spirv_cross_c.cpp

```diff
@@ -699,6 +703,19 @@ spvc_result spvc_compiler_msl_add_resource_binding(spvc_compiler compiler,
 	return SPVC_SUCCESS;
 }
 
+spvc_result spvc_compiler_msl_add_push_descriptor_set(spvc_compiler compiler, unsigned desc_set)
```

Comment:
Can we change this to `add_discrete_descriptor_set()`...as discussed in main conversation?

Reply:
Yes.

Reply:
Renamed now.